src/FormMigration/Steps/FundsAndDesignations.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Give\FormMigration\Steps;
+
+use Give\FormMigration\Contracts\FormMigrationStep;
+use Give\Framework\Blocks\BlockModel;
+
+/**
+ * @unreleased
+ */
+class FundsAndDesignations extends FormMigrationStep
+{
+    /**
+     * @unreleased
+     */
+    public function canHandle(): bool
+    {
+        $fundsAndDesignationsAttributes = $this->getFundsAndDesignationsAttributes($this->formV2->id);
+
+        return count($fundsAndDesignationsAttributes['fund']) !== 0 ||
+               count($fundsAndDesignationsAttributes['options']) !== 0;
+    }
+
+    /**
+     * @unreleased
+     */
+    public function process()
+    {
+        $fundsAndDesignationsAttributes = $this->getFundsAndDesignationsAttributes($this->formV2->id);
+
+        if (count($fundsAndDesignationsAttributes['fund']) === 0 &&
+            count($fundsAndDesignationsAttributes['options']) === 0) {
+            return;
+        }
```

Comment:
since you added the `canHandle` method - this part will be redundant so we can remove.

Reply:
Resolved in 2f8e345096a9133b59690f70fffeb5e9671bb03b